campfire/src/web/serve.rs

```diff
@@ -31,13 +34,19 @@ impl Serve {
                 .context("Failed to run npm install")?;
         }
 
+        self.build
```

Comment:
Ensures the package is always built before running the web server, to avoid situations where you are serving a `pkg` that has been hanging around and wondering why your changes don't show.

That has happened a few of us now 😝